tutorials/adding_visuals.md

```diff
@@ -0,0 +1,117 @@
+\page adding_visuals
+
+# Overview
+
+This tutorial describes how to import 3D meshes into Gazebo to increase the
+visual fidelity of your model. Continuing with our example, we'll attach a 3D
+mesh to our turtle, making it look much better.
+
+## Related tutorials
+
+https://gazebosim.org/api/sim/8/meshtofuel.html
+https://classic.gazebosim.org/tutorials?tut=import_mesh&cat=build_robot
+https://classic.gazebosim.org/tutorials?cat=guided_i&tut=guided_i2
+
+## What is a visual?
+
+The visual element specifies the shape used by the rendering engine. For most
+use cases the collision and visual elements are the same. The most common use
+for different collision and visual elements is to have a simplified collision
+element paired with a visual element that uses a complex mesh. This will help
+improve performance.
+
+SDF supports the notion of visual as described
+[here](http://sdformat.org/spec?ver=1.10&elem=visual).
+
+From our previous tutorial, the turtle visual is a cylinder. Let's use a COLLADA 
+mesh instead. 
+
+## Model directory structure
+
+Gazebo has defined a model directory structure that supports stand-alone models,
+and the ability to share models via an online model database. Review
+[this tutorial](https://gazebosim.org/api/sim/8/meshtofuel.html) for more
+information.
+
+Another benefit of Gazebo's model structure is that it conveniently organizes
+resources, such as mesh files, required by the model.
+
+```bash
+my_turtle
+├── materials               Directory for textures
+    └── textures
+├── meshes                  Directory for COLLADA, STL, and Wavefront OBJ files
```

Comment:
we now also support glTF and glb mesh files. Maybe this can just say `Directory for mesh files such as COLLADA, STL, and Wavefront OBJ files`

Reply:
Changed in [ef9be7b](https://github.com/gazebosim/gz-sim/pull/2257/commits/ef9be7b25be46878ea6a90219cec288e41257826).